### PR TITLE
chore: add eslint to webpack 

### PR DESCRIPTION
**Pull request by delaaxe**

### CODE REVIEW

File: packages/extension/src/background/actionQueue.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import oHash from "object-hash"
-import { InvokeFunctionTransaction, typedData } from "starknet"
 
 import { getFromStorage, setToStorage } from "./storage"
 
+// eslint-disable-next-line @typescript-eslint/ban-types
 function objectHash(obj: {} | null) {
```
Comment (janek26) [inline packages/extension/src/background/actionQueue.ts:6]:
```suggestion
function objectHash(obj: object | null) {
```

File: packages/extension/src/background/actionQueue.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import oHash from "object-hash"
-import { InvokeFunctionTransaction, typedData } from "starknet"
 
 import { getFromStorage, setToStorage } from "./storage"
 
+// eslint-disable-next-line @typescript-eslint/ban-types
```
Comment (janek26) [inline packages/extension/src/background/actionQueue.ts:5]:
```suggestion
```

File: packages/extension/src/background/index.ts [hunk 3]
```diff
@@ -75,7 +75,7 @@ async function main() {
 
   messageStream.subscribe(async ([msg, sender]) => {
     const sendToTabAndUi = async (msg: MessageType) => {
-      sendMessageToActiveTabsAndUi(msg, [sender.tab?.id])
+      sendMessageToActiveTabsAndUi(msg)
```
Comment (janek26) [inline packages/extension/src/background/index.ts:78]: this is needed to send messages to tabs that are **not**:
1. extension UI
2. connected dapps

This means it is needed to answer the sender swell if available, which would be the case for ie extension popups.

**Review by janek26 — CHANGES_REQUESTED** [review]

File: packages/extension/src/background/index.ts (hunk 3 above)
Comment (delaaxe) [reply packages/extension/src/background/index.ts:78]: currently the argument is not used at all. let's add it back when it's actually needed? (no dead code, no "preparation" code on `main`)

Comment (janek26) [reply packages/extension/src/background/index.ts:78]: my fault, it should be used. 
We should add 
```
await sendMessageToActiveTabs(message, additionalTargets)
```
in activeTabs.ts#49

File: packages/extension/src/background/activeTabs.ts [hunk 4]
```diff
@@ -46,5 +46,5 @@ export async function sendMessageToActiveTabsAndUi(
   additionalTargets: Array<number | undefined> = [],
 ) {
   await sendMessageToUi(message)
-  await sendMessageToActiveTabs(message)
+  await sendMessageToActiveTabs(message, additionalTargets)
```
Comment (janek26) [inline packages/extension/src/background/activeTabs.ts:49]: thank you 🙏

Comment (delaaxe) [reply packages/extension/src/background/activeTabs.ts:49]: proof that this is useful :)

**Review by janek26 — APPROVED** [review]

**Pull request merged**